### ISSUE TITLE
Comments and ends of lines

### ISSUE THREAD

**Issue by gadenbuie** [gadenbuie/grkstyle#2]
doesn't seem to like this

```r
base <- c(
  "grep",
  "grepl",
  "sub",  #<<
  "gsub", #<<
  "regexpr",
  "gregexpr",
  "regexec"
)
```

**Comment by lorenzwalthert** [comment]
This marker is detected and preserved by styler as of version 1.4.0 and was resolved in https://github.com/r-lib/styler/pull/700.

**Comment by lorenzwalthert** [comment]
Should you require {styler} 1.4 for this?

**Comment by gadenbuie** [comment]
oh shoot, yeah I should, thanks!
